cli: TestNodeStatus failed under stress

SHA: https://github.com/cockroachdb/cockroach/commits/eb028e6e962b864b79c9d0c610d2c44b26a7d0c4

Parameters:
```
TAGS=
GOFLAGS=-race
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=793289&tab=buildLog

```
=== RUN   TestNodeStatus
W180724 05:40:26.496953 1215 server/status/runtime.go:288  [n?] Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
W180724 05:40:26.843090 1215 sql/jobs/registry.go:299  [n1] unable to get node liveness: node not in the liveness table
W180724 05:40:27.511620 1633 storage/intent_resolver.go:642  [n1,s1] failed to push during intent resolution: failed to push "split" id=84bad8ec key=/Local/Range/System/tsd/RangeDescriptor rw=true pri=0.00847657 iso=SERIALIZABLE stat=PENDING epo=0 ts=1532410827.445586383,0 orig=1532410827.445586383,0 max=1532410827.445586383,0 wto=false rop=false seq=1
W180724 05:40:27.632772 1660 storage/intent_resolver.go:642  [n1,s1] failed to push during intent resolution: failed to push "split" id=eef99743 key=/Local/Range/System/"tse"/RangeDescriptor rw=true pri=0.02913340 iso=SERIALIZABLE stat=PENDING epo=0 ts=1532410827.521904566,0 orig=1532410827.521904566,0 max=1532410827.521904566,0 wto=false rop=false seq=1
W180724 05:40:28.576398 1941 storage/intent_resolver.go:642  [n1,s1] failed to push during intent resolution: failed to push "split" id=0e40cebd key=/Local/Range/Table/21/RangeDescriptor rw=true pri=0.01216540 iso=SERIALIZABLE stat=PENDING epo=0 ts=1532410828.547353821,1 orig=1532410828.515903649,0 max=1532410828.515903649,0 wto=false rop=false seq=1
W180724 05:40:28.874577 1918 storage/intent_resolver.go:642  [n1,s1] failed to push during intent resolution: failed to push "split" id=11e71f6a key=/Local/Range/Table/50/RangeDescriptor rw=true pri=0.00656429 iso=SERIALIZABLE stat=PENDING epo=0 ts=1532410828.839630873,0 orig=1532410828.839630873,0 max=1532410828.839630873,0 wto=false rop=false seq=1
W180724 05:40:28.875989 1903 storage/intent_resolver.go:642  [n1,s1] failed to push during intent resolution: failed to push "split" id=11e71f6a key=/Local/Range/Table/50/RangeDescriptor rw=true pri=0.00656429 iso=SERIALIZABLE stat=PENDING epo=0 ts=1532410828.839630873,0 orig=1532410828.839630873,0 max=1532410828.839630873,0 wto=false rop=false seq=1
[node --insecure=false --certs-dir=/go/src/github.com/cockroachdb/cockroach/artifacts/cli-test891944437 --host=127.0.0.1 --port=41245 status 1 --format=pretty]
[node --insecure=false --certs-dir=/go/src/github.com/cockroachdb/cockroach/artifacts/cli-test891944437 --host=127.0.0.1 --port=41245 status --ranges --format=pretty]
[node --insecure=false --certs-dir=/go/src/github.com/cockroachdb/cockroach/artifacts/cli-test891944437 --host=127.0.0.1 --port=41245 status --stats --format=pretty]
[node --insecure=false --certs-dir=/go/src/github.com/cockroachdb/cockroach/artifacts/cli-test891944437 --host=127.0.0.1 --port=41245 status --ranges --stats --format=pretty]
[node --insecure=false --certs-dir=/go/src/github.com/cockroachdb/cockroach/artifacts/cli-test891944437 --host=127.0.0.1 --port=41245 status --decommission --format=pretty]
[node --insecure=false --certs-dir=/go/src/github.com/cockroachdb/cockroach/artifacts/cli-test891944437 --host=127.0.0.1 --port=41245 status --ranges --stats --decommission --format=pretty]
[node --insecure=false --certs-dir=/go/src/github.com/cockroachdb/cockroach/artifacts/cli-test891944437 --host=127.0.0.1 --port=41245 status --all --format=pretty]
[node --insecure=false --certs-dir=/go/src/github.com/cockroachdb/cockroach/artifacts/cli-test891944437 --host=127.0.0.1 --port=41245 status --format=pretty]
--- FAIL: TestNodeStatus (4.20s)
	cli_test.go:2101: value for key_bytes (30329) greater than max (30000)
	cli_test.go:2101: value for key_bytes (30329) greater than max (30000)
	cli_test.go:2101: value for key_bytes (30329) greater than max (30000)
	cli_test.go:2101: value for key_bytes (30329) greater than max (30000)

```

--- a/pkg/cli/cli_test.go
+++ b/pkg/cli/cli_test.go
@@ -2079,10 +2079,10 @@ func checkNodeStatus(t *testing.T, c cliTest, output string, start time.Time) {
 	if nodeCtx.statusShowStats || nodeCtx.statusShowAll {
 		testcases = append(testcases,
 			testCase{"live_bytes", baseIdx, 100000},
-			testCase{"key_bytes", baseIdx + 1, 30000},
+			testCase{"key_bytes", baseIdx + 1, 50000},
 			testCase{"value_bytes", baseIdx + 2, 100000},
-			testCase{"intent_bytes", baseIdx + 3, 30000},
-			testCase{"system_bytes", baseIdx + 4, 30000},
+			testCase{"intent_bytes", baseIdx + 3, 50000},
+			testCase{"system_bytes", baseIdx + 4, 50000},
 		)
 		baseIdx += len(statusNodesColumnHeadersForStats)
 	}

